src/IssueViz/Helpers/IAnalysisSeverityToVsSeverityConverter.cs

```diff
@@ -48,10 +48,14 @@ public __VSERRORCATEGORY ConvertFromCct(SoftwareQualitySeverity severity)
         {
             switch (severity)
             {
+                case SoftwareQualitySeverity.Blocker:
+                    return __VSERRORCATEGORY.EC_ERROR;
```

Comment:
This should be a warning as well. We don't want to block users at all.

Reply:
Should it?

Reply:
Blocker literally means "error". I think we should clarify this.